.gitignore

```diff
@@ -9,8 +9,6 @@ package-lock.json
 .DS_Store
 
 packages/*/index.js
-apps/*/index.js
-!apps/elections/index.js
```

Comment:
Let's explicitly enumerate the apps that should have their index ignored, instead of special casing the JS ones